pysmt/smtlib/parser/parser.py

```diff
@@ -1181,6 +1239,34 @@ def _cmd_get_value(self, current, tokens):
         self.consume_closing(tokens, current)
         return SmtLibCommand(current, params)
 
+    def _cmd_get_objectives(self, current, tokens):
+        """(get-objective)"""
+        self.parse_atoms(tokens, current, 0)
+        return SmtLibCommand(current, [])
+
+    def _cmd_objective(self, current, tokens):
+        """(maximize | minimize <term> [:id <string>] [:signed] [:lower <const_term>] [:upper <const_term>])"""
+        obj = self.get_expression(tokens)
+        params = []
+        curr = tokens.consume()
+        while curr != ")":
+            tokens.add_extra_token(curr)
+            curr_parse = self.parse_atom(tokens, current)
+            if curr_parse == ":lower" or curr_parse == ":upper":
+                exp = self.get_expression(tokens)
+                params.append((curr_parse, exp))
+            elif curr_parse == ":id":
+                id = self.parse_atom(tokens, "maximization/minimization")
+                params.append((curr_parse, id))
+            elif curr_parse == ":signed":
+                params.append((curr_parse,))
```

Comment:
Minor: Is using the unary tuple the best way of representing this? I did not look at the rest of the code, but `(:signed, True)` and `(:signed, False)` might be nicer from a client perspective.